src/index.js

```diff
@@ -349,7 +362,25 @@ class MiniCssExtractPlugin {
                     'reject(err);',
                   ]),
                   '};',
-                  'linkTag.href = fullhref;',
+                  '} else {',
+                  'var errorTimeout = 60000;',
+                  'var startTime = Date.now();',
+                  'var interval = setInterval(function(){',
+                  'if(linkTag.style){',
```

Comment:
Instead of polling this seem to be a better solution for this problem: https://github.com/webpack-contrib/mini-css-extract-plugin/pull/299/files